### PR TITLE
[fix]Decommission command throws KeeperErrorCode exception when autoRecovery is disabled

### LINKED ISSUE #4489: Decommission command throws KeeperErrorCode exception when autoRecovery is disabled
**BUG REPORT**

***Describe the bug***

When autoRecovery has never been enabled, the autoRecovery component will not be loaded. Audit nodes will not be created on Zookeeper, so an exception will occur when executing Decommission.

***To Reproduce***

1. Deploy a Bookkeeper cluster but do not start the process
2. Disabled AutoRecovery
3. Start the process
4. Execute decommission command

***Expected behavior***
Print `Autorecovery is disabled. So giving up`  instead of Exception

***Screenshots***
QQ_1724591198253

### PR DESCRIPTION
**Pull request by 315157973**
Fix #4489

### Motivation
 Fix decommission command throws KeeperErrorCode exception when autoRecovery is disabled

### CODE REVIEW

**Review by dlg99 — APPROVED** [review]

**Review by shoothzj — APPROVED** [review]

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeperAdmin.java [hunk 1]
```diff
@@ -1553,7 +1553,16 @@ public void triggerAudit()
             throw new UnavailableException("Autorecovery is disabled. So giving up!");
         }
 
-        BookieId auditorId = getLedgerAuditorManager().getCurrentAuditor();
+        BookieId auditorId = null;
+        try {
+            auditorId = getLedgerAuditorManager().getCurrentAuditor();
+        } catch (IOException e) {
+            if (e.getCause() instanceof KeeperException.NoNodeException) {
+                LOG.error("Can not find node for {}", e.getCause().getMessage());
```
Comment (shoothzj) [inline bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeperAdmin.java:1561]:
```suggestion
                LOG.error("Unable to find Zookeeper node: {}", e.getCause().getMessage());
```

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeperAdmin.java [hunk 2]
```diff
@@ -1553,7 +1553,16 @@ public void triggerAudit()
...
+                throw new UnavailableException("Autorecovery is disabled. So giving up!");
```
Comment (shoothzj) [inline bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeperAdmin.java:1562]:
```suggestion
                throw new UnavailableException("Autorecovery is disabled due to missing Zookeeper node. Aborting recovery!");
```

**Review by shoothzj — COMMENTED** [review]
I suggest to modify the error message to be more formal.

**Review by shoothzj — APPROVED** [review]

**Comment by 315157973** [comment]
@dlg99 PTAL

**Pull request merged**